source/less/blocks/form.less

```
@@ -156,16 +156,20 @@
 }
 
 .form__input-radio {
-  &:checked + .form__label::after {
-    display: block;
+  &:checked + .form__label {
```

```suggestion
  .form__input-radio {
    &:checked {
      + .form__label {
        &::after {
          display: block;
        }
      }
    }
  }
```

вот так разбей, 

кстати, а чего у тебя состояние чекед на элементе формы, тогда как оно принадлежит блоку с радио?

---

аналогично для чекбокса

---

ясно, вынесла из формы